server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -2584,6 +2585,53 @@ protected void runInContext() {
         }
     }
 
+    private void verifyVmLimits(UserVmVO vmInstance, Map<String,String> details) {
+        if (VirtualMachineManager.ResoureCountRunningVMsonly.value()) {
```

Comment:
```suggestion
        if (VirtualMachineManager.ResourceCountRunningVMsOnly.value()) {
```

typo ^^^ (not related to this PR)

Reply:
@sureshanaparti hmm, it was introduced by my pr #3760
I did not see it ......